util/pace/util/grid/eta.py

```diff
@@ -35,820 +41,32 @@ def set_hybrid_pressure_coefficients(km: int) -> HybridPressureCoefficients:
-                1.12000002e-04,
-                1.99000002e-04,
-                3.39999999e-04,
-                5.61999972e-04,
-                8.90000025e-04,
-                1.35300006e-03,
-                1.99200003e-03,
-                2.85700010e-03,
-                3.97100020e-03,
-                5.37799997e-03,
-                7.13300006e-03,
-                9.26099997e-03,
-                1.18060000e-02,
-                1.48160001e-02,
-                1.83179993e-02,
-                2.23549996e-02,
-                2.69639995e-02,
-                3.21759991e-02,
-                3.80260013e-02,
-                4.45480011e-02,
-                5.17730005e-02,
-                5.97280003e-02,
-                6.84479997e-02,
-                7.79580027e-02,
-                8.82859975e-02,
-                9.94620025e-02,
-                0.111505002,
-                0.124448001,
-                0.138312995,
-                0.153125003,
-                0.168909997,
-                0.185689002,
-                0.203491002,
-                0.222332999,
-                0.242244005,
-                0.263242006,
-                0.285353988,
-                0.308598012,
-                0.332938999,
-                0.358253986,
-                0.384362996,
-                0.411125004,
-                0.438391000,
-                0.466003001,
-                0.493800014,
-                0.521619022,
-                0.549301028,
-                0.576691985,
-                0.603648007,
-                0.630035996,
-                0.655736029,
-                0.680643022,
-                0.704668999,
-                0.727738976,
-                0.749796987,
-                0.770798028,
-                0.790717006,
-                0.809535980,
-                0.827256024,
-                0.843881011,
-                0.859431982,
-                0.873929024,
-                0.887408018,
-                0.899900019,
-                0.911448002,
-                0.922096014,
-                0.931881011,
-                0.940859973,
-                0.949064016,
-                0.956550002,
-                0.963352025,
-                0.969512999,
-                0.975077987,
-                0.980072021,
-                0.984542012,
-                0.988499999,
-                0.991984010,
-                0.995002985,
-                0.997630000,
-                1.00000000,
-            ]
-        )
-
-    else:
-        raise NotImplementedError(
-            "Only grids with 72, 79, 91 or 137 vertical levels"
-            "have been implemented so far"
-        )
+    (eta_sorted, etav_sorted) = (np.sort(eta), np.sort(etav))
+    for i in range(eta.size):
+        if eta_sorted[i] != eta[i]:
+            raise ValueError("ETA values are not monotonically increasing")
+        if etav_sorted[i] != etav[i]:
+            raise ValueError("ETAV values are not monotonically increasing")
```

Comment:
possible optimization courtesy of numpy slicing:
```suggestion
    if not np.all(eta[:-1] <= eta[1:])
        raise ValueError("ETA values are not monotonically increasing")
    if not np.all(etav[:-1] <= etav[1:])
        raise ValueError("ETAV values are not monotonically increasing")
```